editor/src/components/editor/store/editor-state.ts

```diff
@@ -2913,8 +2913,10 @@ export const DefaultPackageJson = {
   },
 }
 
-export function getPackageJsonFromEditorState(editor: EditorState): Either<string, any> {
-  const packageJsonFile = packageJsonFileFromProjectContents(editor.projectContents)
+export function getPackageJsonFromEditorState(
```

Comment:
```suggestion
export function getPackageJsonFromProjectContents(
```

Reply:
fixed in [getPackageJsonFromProjectContents](https://github.com/concrete-utopia/utopia/pull/4468/commits/324df778c662942fbdd0f04538fd524eeb042a3b)